solr/core/src/java/org/apache/solr/core/NodeConfig.java

```diff
@@ -105,6 +106,8 @@ public class NodeConfig {
 
   private final PluginInfo tracerConfig;
 
+  private final Map<String, String> coreAdminHandlerActions;
```

Comment:
minor: perhaps add after `private final String coreAdminHandlerClass;` since they are related, and similar in other places below

Reply:
No problem, will update it!

Reply:
Done